Habeeb/task_manager.py

```diff
@@ -0,0 +1,85 @@
+import task
+import utils
+
+urgenList=[]
+normalList=[]
+tasksList=[urgenList,normalList]
```

Comment:
I think it is better to use `dict` instead of `array` to easily access the elements when you add, update, delete, and search for an element.